File: Codeforces/C++/problemset/In Search of an Easy Problem.cpp
// Problem Link: https://codeforces.com/problemset/problem/1030/A

#include <iostream>
using namespace std;
typedef long long ll;
int main()
{
    ll n,p,g = 0;
    cin >> n;
    for(ll i = 0; i < n; i++)
    {
        cin >> p;
        if(p == 1) g++;
    }
    (g == 0) ? cout << "EASY" : cout << "HARD";
    return 0;
}

File: Codeforces/Kotlin/src/main/kotlin/problems/InSearchOfAnEasyProblem.kt
package problems.problems

/* Problem Link: https://codeforces.com/problemset/problem/1030/A */

fun main() {
    val n = readln()
    val opinions = readln().split(" ")

    print(if (opinions.count { it == "1" } > 0) "HARD" else "EASY")
}